op-node/cmd/genesis/cmd.go

```diff
@@ -223,3 +270,55 @@ func writeGenesisFile(outfile string, input any) error {
 	enc.SetIndent("", "  ")
 	return enc.Encode(input)
 }
+
+// rpcBlock represents the JSON serialization of a block from an Ethereum RPC.
```

Comment:
To make sure I follow, we define our own serialization structs here rather than importing them because the RPC could really be coming from anywhere, right?

Reply:
geth defines them in an `internal` package so we cannot import them unfortunately